docs/GraFx-Studio/guides/paragraphstyles/index.md

```diff
@@ -2,26 +2,28 @@
 
 ## Define a paragraph style
 
-Open the Stylekit panel.
+Open the **Resource Panel** > **Stylekit panel**.
```

Comment:
I noticed the Character Styles page didn't get the update with the new Resources panel.